.dockerignore

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
Why is this needed? I'd like to avoid polluting this repo with a bunch of docker-specific stuff.

Reply:
Omitting the README from the build context makes development faster? How so? Cache invalidation?